src/components/employee/employee.service.js

```
@@ -125,6 +131,9 @@ module.exports.update = async (id, employee) => {
         if (!employee)
             throw new error.BadRequest('employee data not provided');
 
+        if (!validateDates(employee.vacationStartDate, employee.vacationEndDate) && employee.vacationStartDate && employee.vacationEndDate)
```

This validation is incorrect. Remember that the vacation dates are **not required**. If they're null, we simply set them as null on the employee document.

---

@pawap90 You are right, the reason i made that is because i didnt want to throw a BadRequest error message if the vacation dates are missing.

the way its coded, a BadRequest will be thrown when both dates exist, and the two dates are not valid.

Is there anything wrong?